### PR TITLE
configure.sh: stylistic improvements

### PR DESCRIPTION
**Pull request by vitorgalvao**
Making the script more consistent with itself, as well as fixing some possible pitfalls.

### CODE REVIEW

File: configure/configure.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```
Comment (vitorgalvao) [inline configure/configure.sh]: No reason not to use `/bin/bash` instead of `/bin/sh`, on macOS.

File: configure/configure.sh [hunk 2]
```diff
@@ -10,29 +10,29 @@
...
 
 
-if [ "$1" == "-install" ]
+if [ "${1}" == "-install" ]
 then
     echo "installing"
 
     #move to unzipped directory
-    cd "`dirname "$0"`"
+    cd "$(dirname "${0}")" || exit -2
```
Comment (vitorgalvao) [inline configure/configure.sh]: The `||` is a [`shellcheck`](https://github.com/koalaman/shellcheck) recommendation. It prevents problems if the `cd` can’t be done (extremely unlikely, but no reason to not do it).

File: configure/configure.sh [hunk 3]
```diff
@@ -10,29 +10,29 @@
...
 
     #remove all xattrs
-    /usr/bin/xattr -rc *
+    /usr/bin/xattr -rc ./*
```
Comment (vitorgalvao) [inline configure/configure.sh]: In case there ever are files starting with `-`, this avoids treating them as flags.

File: configure/configure.sh [hunk 4]
```diff
@@ -65,11 +65,10 @@ then
 
     echo "install complete"
 
-exit 1
```
Comment (vitorgalvao) [inline configure/configure.sh]: Why does a successful install have a non-zero exit code?

File: configure/configure.sh [hunk 5]
```diff
@@ -87,20 +86,18 @@ then
     rm -rf /Library/Objective-See/LuLu/
 
     #kill
-    killall "LuLu" 2> /dev/null
-    killall "com.objective-see.luluHelper" 2> /dev/null
+    killall LuLu 2> /dev/null
```
Comment (vitorgalvao) [inline configure/configure.sh]: Removed quotes for consistency with other parts of the script.

**Comment by objective-see** [comment]
@vitorgalvao mahalo for improvements 👍  Can you sign the CLA, then will merge the PR :)

**Comment by vitorgalvao** [comment]
Signed.

**Pull request merged**